src/components/projects/view/moderator-hour-card.svelte

```diff
@@ -45,11 +45,12 @@
         />
         {#if enteredHours !== application.actual_hours}
           <Button
-            isRound
+            isFilled
             disabled={enteredHours < 0}
             on:click={dispatchHourChange}
           >
-            <svg class="icon" src="images/icons/check.svg" />
+            <svg class="icon" src="images/icons/save.svg" />
```

Comment:
This should probably be `class="icon mr"` then

Reply:
I removed it on purpose as it was too long, but I will put it back since I will remove the word "hours"